presto-benchto-benchmarks/README.md

```diff
@@ -62,6 +62,17 @@ macros:
       
 ```
 
+### Bootstrapping benchmark data
+* Configure Presto [TCP-H connector](https://prestosql.io/docs/current/connector/tpch.html), if you haven't.
+* Generate SQL file:
+```bash
+python presto-benchto-benchmarks/generate_schemas/generate-tpch.py tpch.sql
```

Comment:
nit: It should be possible to pipe this directly to presto-cli

Reply:
good idea!